_posts/blog/2017-05-02-transitioning-data-pipeline.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Transitioning to a New Backend Pipeline and Data Availability"
+author: "Richard Roberto"
+breadcrumb: blog
+categories:
+  - data
+  - pipeline
```

Comment:
I was just looking at some of our existing tags, let's add:
- bigquery
- research
- data-analysis
- gcs
- performance